dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -240,17 +268,17 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
             if (isProtocolActivatorSet)
```

Comment:
It would be nice when all this stuff is cleaned away by @sharath2727 . It is way too complex in its current form and I'm looking forward to the cleanup.